### PR TITLE
Working submit and confirmation page for 1995

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [939](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/939) and [1095](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1095)

Adds the code to transform the form into the right format for the backend. Also updates confirmation page with correct info and design changes from @goldenmeanie.

screen shot 2017-02-07 at 11 40 39 am

### CODE REVIEW

**Comment by goldenmeanie** [comment]
The plus sign in that accordion looks smaller than usual, right?

**Comment by goldenmeanie** [comment]
"Benefit claimed" might be better as "Benefit to be transferred"

**Comment by jbalboni** [comment]
@goldenmeanie Yeah, not sure why the accordion styles are weird on this page. Same issue on 1990.

**Comment by goldenmeanie** [comment]
Is there some sort of nested 80% thing going on?

File: src/js/common/schemaform/helpers.js [hunk 1]
```diff
@@ -156,13 +158,63 @@ export function parseISODate(dateString) {
...
+ * Normal transform for schemaform data
+ */
+export function transformForSubmit(formConfig, form) {
+  const activePages = getActivePages(createFormPageList(formConfig), form);
+  const flattened = flattenFormData(activePages, form);
+  const withoutViewFields = filterViewFields(flattened);
+
+  return JSON.stringify(withoutViewFields, (key, value) => {
+    // an object with country is an address
+    if (value && typeof value.country !== 'undefined' &&
```
Comment (annekainicUSDS) [inline src/js/common/schemaform/helpers.js]: Is this because country is defaulted to "USA"?

**Review by annekainicUSDS — APPROVED** [review]
Looks great and tested great too!

File: src/js/common/schemaform/helpers.js (hunk 1 above)
Comment (jbalboni) [inline src/js/common/schemaform/helpers.js]: Yeah, otherwise it'd be an empty object by default and we could just remove it.

**Pull request merged**